static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -33,64 +34,70 @@ define([
     onSlotRender,
     onSlotLoad,
     PrebidService,
-    ophanTracking
+    prepareSonobiTag,
+    performanceLogging
 ) {
-    return init;
+    return {
+        init: init,
+        customTiming: true
+    };
 
-    function init() {
+    function init(moduleName) {
 
-        var removeAdSlots = function () {
+        function removeAdSlots() {
             bonzo(qwery(dfpEnv.adSlotSelector)).remove();
-        };
+        }
 
-        if (commercialFeatures.dfpAdvertising) {
-            var initialTag = dfpEnv.sonobiEnabled ? setupSonobi() : Promise.resolve();
-            return initialTag.then(setupAdvertising).catch(function(){
-                // A promise error here, from a failed module load,
-                // could be a network problem or an intercepted request.
-                // Abandon the init sequence.
-                return fastdom.write(removeAdSlots);
-            });
+        function moduleCompleted() {
+            performanceLogging.moduleEnd(moduleName);
         }
 
-        return fastdom.write(removeAdSlots);
-    }
+        function setupAdvertising() {
+            // Use Custom Timing to time the googletag code without the sonobi pre-loading.
+            performanceLogging.moduleStart(moduleName);
 
-    function setupSonobi() {
-        return new Promise.resolve(require(['js!sonobi.js']));
-    }
+            return new Promise(function(resolve) {
 
-    function setupAdvertising() {
+                if (dfpEnv.sonobiEnabled) {
+                    // Just load googletag. Sonobi's wrapper will already be loaded, and googletag is already added to the window by sonobi.
+                    require(['js!googletag.js']);
+                    performanceLogging.addTag('sonobi');
+                } else {
+                    require(['js!googletag.js']);
 
-        return new Promise(function(resolve) {
+                    if (dfpEnv.prebidEnabled) {
+                        dfpEnv.prebidService = new PrebidService();
+                        performanceLogging.addTag('prebid');
+                    } else {
+                        performanceLogging.addTag('waterfall');
+                    }
+                }
 
-            if (dfpEnv.sonobiEnabled) {
-                // Just load googletag. Sonobi's wrapper will already be loaded, and googletag is already added to the window by sonobi.
-                require(['js!googletag.js']);
-                ophanTracking.addTag('sonobi');
-            } else {
-                require(['js!googletag.js']);
+                window.googletag.cmd.push = raven.wrap({deep: true}, window.googletag.cmd.push);
```

Comment:
I've always wondered whether this line made any sense at all. When `googletag.js` loads, it overwrites `googletag.cmd.push`. Anyway, that's totally unrelated 😆